tools/cdk-build-tools/config/jest.config.json

```diff
@@ -18,4 +23,4 @@
   "coveragePathIgnorePatterns": [
     "<rootDir>/lib/.*\\.generated.js"
```

Comment:
We should also ignore coverage in `<rootDir>/test`

Reply:
I believe you get that for free...? (tests are always excluded for coverage, unless I'm missing something)

Reply:
You were right - tests are, but files inside the test directories (like testing utilities) are not - great catch! Will fix.